### ISSUE TITLE
fix(general firefox): set system-color keywords Canvas, CanvasText, Highlight, ButtonBorder

### ISSUE THREAD

**Issue by LarissaMS** [swisspost/design-system#340]
- [ ] Focus outline should be set to 'Highlight'
- [ ] Text color should be set to 'CanvasText'
- [ ] Background color should be set to 'Canvas'
- [ ] Set 'ButtonBorder'

https://www.w3.org/TR/css-color-4/#css-system-colors:~:text=defined%20as%20follows%3A-,Canvas,-Background%20of%20application

**Comment by gfellerph** [comment]
Use CSS Variables to override default styles on all components. This means preparing all the components accordingly. Separate tasks for this need to be set up.

**Comment by gfellerph** [comment]
Closing as being implemented (in individual components)
